### PR TITLE
Add Karma example

### PR DESCRIPTION
**Pull request by Robdel12**
## What is this?

This adds Karma as an example that lives alongside the BigTest CLI.

### CODE REVIEW

File: react/countdown-karma/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+const convert = require('koa-connect');
+const history = require('connect-history-api-fallback');
+
+const isTesting = process.env.NODE_ENV === 'testing';
```
Comment (Robdel12) [inline react/countdown-karma/webpack.config.js]: This probably isn't needed. Should I delete it?

Comment (wwilsman) [reply react/countdown-karma/webpack.config.js]: Yea

**Review by wwilsman — APPROVED** [review]
I'm going to just assume we rolled back changes from the other example and spun it off into another directory. In which case I'm going to opt out of reviewing code I technically wrote.

**Comment by Robdel12** [comment]
@wwilsman Yep! hhaha that's what happened. I'm not _crazy_ about maintaining two webpack configs and what not, but I don't think it's going to be that big of a deal.

**Pull request merged**